app/assets/stylesheets/govuk_publishing_components/components/_table.scss

```
@@ -37,28 +37,26 @@ $table-row-even-background-colour: govuk-colour("light-grey", $legacy: "grey-4")
       padding-right: $sort-link-arrow-size;
       color: $govuk-link-colour;
       text-decoration: none;
+    }
 
-      &:focus {
-        @include govuk-focused-text;
-      }
-
-      &::after {
-        content: "";
-        position: absolute;
-        top: 5px;
-        right: 0;
-
-        @include govuk-shape-arrow($direction: up, $base: $sort-link-arrow-size-small, $display: block);
-      }
+    .app-table__sort-link &:focus {
+      @include govuk-focused-text;
+    }
 
-      &::before {
-        content: "";
-        position: absolute;
-        top: 13px;
-        right: 0;
+    .app-table__sort-link &:after {
```

I suggest we simplify this:
```suggestion
    .app-table__sort-link:after {
```